EmployeeManagement/Tavisca.EmployeeManegent.ServiceLibrary/IEmployeeService.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.ServiceModel;
+using System.Text;
+using System.Threading.Tasks;
+using System.ServiceModel.Web; 
+using Tavisca.EmployeeManagement.DataContract;
+
+namespace Tavisca.EmployeeManagement.ServiceContract
+{
+    [ServiceContract]
+    public interface IEmployeeService
+    {
+        [WebInvoke(Method = "GET", UriTemplate = "/get_record/{id}", ResponseFormat = WebMessageFormat.Json)]
```

Comment:
don't use '_'in the uri templates.  Also as per rest guidelines the url template should be "/employee/{id}" and "/employee"